src/components/users/login/LoginPage.js

```diff
@@ -38,19 +52,24 @@ const LoginPage = () => {
   };
 
   return (
-    <div className='login-page'>
+    <div className="login-page">
       <div>
         <h1>Cohort Manager 2.0</h1>
       </div>
-      <Link id='user-registration-link' to='/signup'>
+      <Link id="user-registration-link" to="/signup">
         sign up!
       </Link>
-      <Link id='user-login-link' to='/'>
+      <Link id="user-login-link" to="/">
         login!
       </Link>
       <h1>Login</h1>
       <p>Status: {loginResponse.status}</p>
       <UserForm handleChange={handleChange} handleSubmit={loginUser} />
+      {loginError ? (
+        <Alert severity="error">Email or Password is incorrect</Alert>
+      ) : (
+        <></>
+      )}
```

Comment:
Try { loginError && (...)} I think that should work and you won't need the ':' else stuff